packages/cli/src/options/heap/AnalysisOutputOption.ts

```diff
@@ -34,7 +34,7 @@ export default class AnalysisOutputOption extends BaseOption {
   ): Promise<void> {
     const name = this.getOptionName();
     config.outputFormat = AnalysisOutputOption.parseOutputFormat(args[name]);
```

Comment:
if user does not provide a `--output` flag, `args[name]` is `undefined`, it will cause an exception in `parseOutputFormat `